cf-macro/template.yaml

```diff
@@ -0,0 +1,31 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: AWS::Serverless-2016-10-31
+Metadata:
+  AWS::ServerlessRepo::Application:
+    Name: cf-macro-app
+    Description: Creates a Macro that can auto-generate Cloudwatch Alarms and Dashboard.
+    Author: FourTheorem
+    SpdxLicenseId: Apache-2.0
```

Comment:
I think that `SpdxLicenseId` is a unique identifier for our application, like `SlicWatch`